test/Microsoft.AspNet.Mvc.Core.Test/ControllerTests.cs

```diff
@@ -374,6 +374,139 @@ public void RedirectToRoutePermanent_WithParameterRouteName_SetsResultSameRouteN
         }
 
         [Fact]
+        public void Created_WithStringParameter_SetsCreatedLocation()
+        {
+            // Arrange
+            var controller = new Controller();
+            var uri = "http://test/url";
+
+            // Act
+            var result = controller.Created(uri, null);
+
+            // Assert
+            Assert.IsType<CreatedResult>(result);
```

Comment:
nitpick: these assert.types doesn't really do anything - `result` is already the type you want. Sure, you can still verify the type, but the details we **really** care about are things like `.Location`